src/app/User.test.ts

```diff
@@ -58,20 +60,21 @@ describe(UserService.name, () => {
     it("Should pass the right data to db", async () => {
       // Arrange
       const date = new Date("Date Mon Oct 24 2022 17:21:08 GMT-0300");
-      const expectedValue = "2022-10-24T20:21:08.000Z";
```

Comment:
Removing redundant code is good, but it would be nice do it in another commit.

Reply:
I think that this code should not be removed. In any case, it should be part of another commit because it is not part of "update and refactor to aws sdk v3"

Reply:
👍